src/applications/vaos/appointment-list/pages/PastAppointmentsPage/index.unit.spec.jsx

```diff
@@ -53,11 +53,15 @@ describe('VAOS Page: PastAppointmentsList api', () => {
       statuses: ['booked', 'arrived', 'fulfilled', 'cancelled'],
     });
 
-    const { findByText } = renderWithStoreAndRouter(<PastAppointmentsList />, {
+    const screen = renderWithStoreAndRouter(<PastAppointmentsList />, {
       initialState,
     });
 
-    expect(await findByText(/Past 3 months/i)).to.exist;
+    await screen.findByText(/We didn’t find any results in this date range/i);
+
+    expect(
+      await screen.container.querySelector('va-select[name="date-dropdown"]'),
```

Comment:
Can we add a unit test for when the dropdown is not visible? I.e. when the state is loading or not started?